src/frontend/src/hooks/projects.hooks.ts

```diff
@@ -166,3 +167,21 @@ export const useCreateLinkType = () => {
     }
   );
 };
+
+/**
+ * Custom React Hook to edit a LinkType.
+ *
+ * @param linkTypeName The name of the LinkType to edit (unique)
+ * @returns the edited linkType
+ */
+export const useEditLinkType = (linkTypeName: string) => {
+  const queryClient = useQueryClient();
+  return useMutation<LinkType, Error, LinkTypeCreatePayload>(
+    ['linkTypes', 'edit'],
+    async (formData: LinkTypeCreatePayload) => {
+      const { data } = await editLinkType(linkTypeName, formData);
+      queryClient.invalidateQueries(['linkTypes']);
```

Comment:
this should be happening in the onSuccess callback (see how we do it in other hooks)